src/main/java/de/qabel/desktop/ui/actionlog/ActionlogController.java

```diff
@@ -21,75 +31,143 @@
 
 	public void initialize(URL location, ResourceBundle resources) {
+
+		identity = clientConfiguration.getSelectedIdentity();
+		c = new Contact(identity, identity.getAlias(), identity.getDropUrls(), identity.getEcPublicKey());
 		try {
-			loadMessages();
+			loadMessages(c);
 		} catch (EntityNotFoundExcepion entityNotFoundExcepion) {
 			entityNotFoundExcepion.printStackTrace();
 		}
+
+		scroller.setVvalue(scroller.getVmax());
+		addListener();
 	}
 
-	void loadMessages() throws EntityNotFoundExcepion {
-		messages.getChildren().clear();
-		createMessageItems();
-		if(scroller.getVmax() == scroller.getVvalue()) {
-			scroller.setVvalue(scroller.getVmax());
+	private void addListener() {
+		((Region) scroller.getContent()).heightProperty().addListener((ov, old_val, new_val) -> {
+			if (scroller.getVvalue() != scroller.getVmax()) {
+				scroller.setVvalue(scroller.getVmax());
+			}
+		});
+		textarea.setOnKeyPressed(keyEvent -> {
+			if (keyEvent.getCode().equals(KeyCode.ENTER) && keyEvent.isControlDown()) {
+
+				try {
+					sendDropMessage(c, textarea.getText());
+					receiveDropMessages(lastDate);
+					loadMessages(c);
+
+				} catch (QblDropPayloadSizeException | PersistenceException | QblDropInvalidMessageSizeException | QblVersionMismatchException | QblSpoofedSenderException | QblNetworkInvalidResponseException | EntityNotFoundExcepion e) {
+					e.printStackTrace();
+				}
+			}
+		});
+	}
+
+	@FXML
+	protected void handleSubmitButtonAction(ActionEvent event) throws QblDropPayloadSizeException, EntityNotFoundExcepion, PersistenceException, QblDropInvalidMessageSizeException, QblVersionMismatchException, QblSpoofedSenderException, QblNetworkInvalidResponseException {
+		if (textarea.getText() == "") {
+			return;
 		}
+		sendDropMessage(c, textarea.getText());
+		receiveDropMessages(lastDate);
+		loadMessages(c);
 	}
 
-	private void createMessageItems() {
-		String text;
-		Map<String, Object> injectionContext = new HashMap<>();
-		text = "Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.</html>";
-		injectionContext.put("text", text);
-		MyActionlogItemView myItemView = new MyActionlogItemView(injectionContext::get);
-		messages.getChildren().add(myItemView.getView());
-		messageView.add(myItemView);
+	void receiveDropMessages(Date siceDate) throws QblDropInvalidMessageSizeException, QblVersionMismatchException, QblSpoofedSenderException, PersistenceException, EntityNotFoundExcepion {
```

Comment:
I don't think this should be handled by the controller. Even without a GUI, DropMessages needs to be received so it belongs to some kind of background daemon handle new drop messages.